UPGRADE.md

```diff
@@ -0,0 +1,23 @@
+# Upgrading Phragile
+
+When upgrading Phragile to a newer version **always** run
+
+    composer install
+    php artisan migrate
+
+after updating the files in the root directory of your application.
+
+## From 1.1 release to 2.0.0
+
+### Migrate your Snapshots
+
+To support newer versions of Phabricator, Phragile 2.x.x uses manifest.search instead of manifest.query requests. Therefore older snapshot data needs to be migrated with
```

Comment:
Oh my, I might have read the whole sentence before making a comment, sorry!
"Since 2.0.0" sounds reasonable.